README.md

```diff
@@ -9,7 +9,7 @@
 [docs-badge]: https://docs.rs/tinychain/badge.svg
 [docs-url]: https://docs.rs/tinychain/
 [discord-badge]: https://img.shields.io/discord/909338776675967046.svg
-[discord-url]: https://discord.com/channels/909338776675967046
+[discord-url]: https://discord.gg/jQMnN5k3D9
```

Comment:
I think you might want to do a rebase so these intermediary changes don't cause any merge conflicts with the main branch